srcpkgs/azote/template

```diff
@@ -10,15 +10,14 @@ maintainer="Piotr Miller <nwg.piotr@gmail.com>"
 license="BSD-3-Clause, GPL-3.0-or-later"
 homepage="https://github.com/nwg-piotr/azote"
 distfiles="${homepage}/archive/v${version}.tar.gz"
-checksum=e9ecd2104689c05fb2a37bb160543d80e9fd245c5274ab16fc021f400d738d35
+checksum=d8b2f5f4e351e72d9c58f9d5fc9c9e2526087e57e5553f6defbd163c78727905
 
 post_install() {
 	vmkdir usr/bin
 	vmkdir usr/share/${pkgname}
 	vmkdir usr/share/applications
 
 	vcopy "dist/${pkgname}.svg" usr/share/${pkgname}
-	vcopy "dist/${pkgname}-arch" usr/share/${pkgname}
```

Comment:
`azote-arch` is removed from upstream: https://github.com/nwg-piotr/azote/commit/23c974a4bf7d578b8783f4032c57e6e6355fbc29

Reply:
True, but now /usr/bin/azote is broken symlink.

Reply:
This is embarrassing, totally overlooked the symlink. Fixed now.